NEWS.md

```diff
@@ -1,61 +1,71 @@
-# scales 0.5.0.9000
+# scales 1.0.0
 
-* `expand_range()` arguments `mul` and `add` now affect scales with a range of 0
-   (@dpseidel, tidyverse/ggplot2#2281).
+## New Features
 
-* `date_breaks()` now supports subsecond intervals (@dpseidel, #85).
-
-* New function `modulus_trans()` implements the modulus transformation for positive
-  and negative values (@dpseidel).
-
-* `boxcox_trans()` is now invertible for `x >= 0` and requires positive values 
-  (@dpseidel, #103).
+### Formatters
 
-* New functions `byte_format()` and `bytes()` format numeric vectors into byte
+* New `byte_format()` and `bytes()` format numeric vectors into byte
   measurements (@hrbrmstr, @dpseidel).
+  
+* `comma_format()`, `percent_format()` and `unit_format()` gain new arguments: 
+  `accuracy`, `scale`, `prefix`, `suffix`, `decimal.mark`, `big.mark` 
+  (@larmarange, #146).
 
-* `extended_breaks()` no longer forces `labeling::extended` argument
-  `only.loose = FALSE` to allow for more flexible breaks specification 
-  (@dpseidel, #99).
+* `dollar_format()` gains new arguments: `accuracy`, `scale`, `decimal.mark`, 
+  `trim` (@larmarange, #148).
+
+* New `number_format()` provides a generic formatter for numbers (@larmarange, #142).
+  
+* New `pvalue_format()` formats p-values (@larmarange, #145).
 
-* Remove `plyr` and `dichromat` dependencies. `dichromat` is now suggested
- (@dpseidel, #118).  
+* `ordinal_format()` gains new arguments: `prefix`, `suffix`, `big.mark`, `rules`,
+   and rules for French and Spanish are also provided (@larmarange, #149).
```

Comment:
I'd switch a semi-colon in for the note at the end ⇨ "`rules`; rules for French and Spanish..."